arraycontext/loopy.py

```diff
@@ -70,5 +70,47 @@ def get_default_entrypoint(t_unit):
 
 # }}}
 
-
-# vim: foldmethod=marker
+class BaseFakeNumpyNamespace:
```

Comment:
Rename this to `LoopyBasedFakeNumpyNamespace`.

Reply:
Renamed !!